storage/google_cloud/google_cloud_utils.py

```diff
@@ -88,5 +90,28 @@ def upload_file_to_blob(bucket_credentials_file_path, target_blob_url, f):
     log.info(f"Uploading file to blob '{target_blob_url}'...")
     storage_client = storage.Client.from_service_account_json(bucket_credentials_file_path)
     blob = _blob_at_url(storage_client, target_blob_url)
-    blob.upload_from_file(f)
-    log.info(f"Uploaded file to blob")
+
+    try:
+        blob.upload_from_file(f)
+        log.info(f"Uploaded file to blob")
+
+    except HttpError or socket.timeout as ex:
```

Comment:
I often see requests.exceptions.ConnectionError. Is that a subclass of one of these errors?

Reply:
Have added `ConnectionError, Timeout` which are the most common errors we encounter. They are part of request exceptions: https://2.python-requests.org/en/master/_modules/requests/exceptions/